firefox_puppeteer/__init__.py

```diff
@@ -48,6 +40,14 @@ def keys(self):
         See the :class:`~api.keys.Keys` reference.
         """
 
+    @property
+    def platform(self):
+        """Returns the lowercased platform name.
+
+        :returns: Platform name
+        """
+        return self.marionette.session_capabilities['platformName'].lower()
```

Comment:
I thought about this first too but abandoned this idea because it would not be available for sub modules in that package. Something I would like to have is that modules would be able to access API classes and properties like those. Therefor the puppeteer modules need to be refactored so it only holds API level stuff. I would like to do that later.